rbd/mirror_test.go

```diff
@@ -769,7 +771,6 @@ func TestMirrorBootstrapToken(t *testing.T) {
 		err = SetMirrorMode(ioctx2, MirrorModeImage)
 		require.NoError(t, err)
 
-		fmt.Printf("TOKEN: %s\n", string(token))
```

Comment:
I hope this change is not related to new API.

Reply:
Nope, I just cleaned a few things up I noticed while running the tests.  Did you see that it's in a separate commit titled `rbd: remove stray debugging prints from mirror tests`?

Reply:
Ah..never bothered to look into commits 😶‍🌫️ 

Fine.